calibrators/marker_radar_lidar_calibrator/README.md

```diff
@@ -0,0 +1,168 @@
+# marker_radar_lidar_calibrator
+
+A tutorial for this calibrator can be found [here](../docs/tutorials/marker_radar_lidar_calibrator.md)
+
+## Purpose
+
+The package `marker_radar_lidar_calibrator` allows extrinsic calibration between radar and 3d lidar sensors used in autonomous driving and robotics.
+
+Currently, the calibrator only supports radars whose detection interface includes distance and azimuth angle, but do not offer elevation angle. For example, ARS 408 radar can be calibrated with this tool. Also, note that the 3d lidar should have a resolution that is high enough to scan several points on the [radar reflector](#radar-reflector) (calibration target).
+
+## Inner-workings / Algorithms
+
+The calibrator is designed to estimate the transformation between radar and lidar sensors. It starts by pinpointing the central points of reflectors within lidar pointclouds and radar objects, then aligns these points for precise matching. An SVD-based and a yaw-only rotation estimation algorithm are applied to these matched points to estimate the transformation.
+
+Specifically, the calibration process consists of four primary steps: constructing a background model, extracting the foreground to detect reflectors, matching and filtering lidar and radar detections, and finally executing the calibration process.
+
+### Step 1: Background model construction
+
+Given the challenge of reliably detecting reflectors, such as when radar outputs numerous objects making it difficult to identify the reflector among them, background models for both lidar and radar are constructed from the lidar pointclouds and radar objects within a user-defined calibration area, which lacks any calibration targets (such as radar reflectors). More specifically, these background models consist of uniform binary voxel grids that denote whether each voxel represents the background.
+
+### Step 2: Foreground extraction and reflector detection
+
+After the background models for the lidar and radar are established, we extract the foreground points from incoming lidar pointclouds and radar objects that are not in the background voxels. All foreground radar points are automatically categorized as potential reflector detections.
```

Comment:
Establish is not the right verb here

Reply:
The explanation of foreground points is logically incorrect.
You compare the input pointcloud with the background model, extract those that not belong to it, and call them foreground points

Reply:
So far, radar information have been called objects or detections. From there on they are sometimes denoted as points / pointclouds. Keep consistency with the object / detection nomenclature. If needed, define the differences between both

Reply:
fixedi in https://github.com/tier4/CalibrationTools/pull/164/commits/8a011a534e4a7c6cd98903379d8f95f3cd198138